vnext/ReactUWP/Views/FlyoutViewManager.cpp

```diff
@@ -182,6 +183,12 @@ void FlyoutShadowNode::updateProperties(const folly::dynamic&& props)
         m_isLightDismissEnabled = propertyValue.asBool();
       else if (propertyValue.isNull())
         m_isLightDismissEnabled = true;
+      if (m_targetElement != nullptr)
```

Comment:
Ashita suggested that we use m_isOpen as the marker for an open flyout rather than the m_targetElement.